Daljeet/downtimes v2 main docs (#19154)
DataDog/documentation: content/en/monitors/downtimes/_index.md

You would like to make sure that `service:web-store` related notifications are muted and that other critical alerts for the remaining services are delivered as usual. For that, simply enter `service:web-store` in the Downtime's group scope after selecting the monitor targets.

**Note**: this approach also works with groups that have multiple dimensions, e.g. `service` and `host`. Creating a Downtime on `service:web-store` would mute all groups that include said service, e.g. `service:web-store,host:a` or `service:web-store,host:b`.

#### Scoping on monitor query filter
It is common to filter a monitor query to only look at dimensions you care about. You can create Downtimes that target those same dimensions. This approach doesn't require you to add additional grouping to a monitor where it wouldn't really be necessary.

{{< img src="/monitors/downtimes/downtime_scope_query.png" alt="Example of a monitor's query filter" style="width:100%;" >}}

The monitor above will be muted by a Downtime that matches the monitor specific target and is scoped by `env:prod`.

#### Downtime scope syntax

> The Downtime scope query follows the same common [Search Syntax][19] that many other products across the platform support. To include all groups in the scope of a Downtime, type `*` for the `Group scope`. Further examples of group scopes include:

Author: It's definitely closer to the Logs syntax (monitor search has some specific nuances that are a bit unique). I'll add more details in your other comment about the syntax!

| `env:(dev OR staging)`       | Mutes any notification related to either the `dev` or `staging` environment. |
| `service:web-store AND env:(dev OR staging)`       | Mutes any notification related to the `web-store` service running in either the `dev` or `staging` environment. |
| `host:authentication-*`       | Mutes any notification that relates to a host whose name is prefixed with `authentication-`. |
| `host:*-prod-cluster`       | Mutes any notification that relates to a host whose name is suffixed with `-prod-cluster`. |
| `host:*-prod-cluster`       | Mutes any notification that relates to a host whose name is suffixed with `-prod-cluster`. |
| `service:webstore AND -env:prod`       | Mutes any notification about the `web-store` service that is **not** running on the `prod` environment. |

#### Downtime scope limitations
There are a few limitations that are **not supported** which include:

* More than two levels of nesting, e.g. `team:app AND (service:auth OR (service:graphics-writer AND (env:prod OR (type:metric AND status:ok))))`, are not supported. At most, Downtimes accept two levels of nesting. Use separate Downtimes instead to break down the logic.

> * Negation is only supported for key/value pairs and tags with `OR`, e.g. `-key:value` and `-key(A OR B)`. Scopes such as `-service:(A AND B)`, `service:(-A OR -B)`, `-service(A B)`, etc. are not supported.

Reviewer: ```suggestion
* Negation is only supported for key/value pairs and tags with `OR`. For example, `-key:value` and `-key(A OR B)`. Scopes such as `-service:(A AND B)`, `service:(-A OR -B)`, or `-service(A B)` are not supported.
```

> * More than two levels of nesting, e.g. `team:app AND (service:auth OR (service:graphics-writer AND (env:prod OR (type:metric AND status:ok))))`, are not supported. At most, Downtimes accept two levels of nesting. Use separate Downtimes instead to break down the logic.

Reviewer: Is this a common complaint with users, not knowing search limitations? We don't typically surface what we don't support. Is this something we have plans to support in the future?

> * Top level ORs are not supported, e.g. `service:A OR host:X`. This requires two separate Downtimes.

Reviewer: ```suggestion
* Top level ORs are not supported, for example, `service:A OR host:X`. This requires two separate Downtimes.
```

> * Keyless tags, e.g. `prod AND service:(A or B)` or just `prod`, aren't supported. Tags need to have a key, in this case for example `env:prod`.

Reviewer: ```suggestion
* Keyless tags, such as `prod AND service:(A or B)` or just `prod`, aren't supported. Tags need to have a key, in this case for example `env:prod`.
```

[16]: /integrations/amazon_ec2/#ec2-automuting
[17]: /integrations/google_compute_engine/#gce-automuting
[18]: /integrations/azure_vm/#automuting-monitors

> [19]: /logs/explorer/search_syntax/

Reviewer: Would this link make more sense here or are these different?
https://docs.datadoghq.com/monitors/manage/search/
